ap_versioned_docs/version-2.9.0/admin-guide/getting-started.mdx

```diff
@@ -0,0 +1,384 @@
+---
+title: "Getting Started"
+sidebar_position: 30
+---
+
+## Installation
+
+import { CodeExample } from "@site/src/components/CodeExample";
+
+The easiest way to install the Anchor Platform is to pull the [docker image][anchor-platform-image].
+
+<CodeExample>
+
+```bash
+docker pull stellar/anchor-platform:latest
```

Comment:
We still pill `latest` here, I think we should start pinning versions from now on. Maybe do replacement of 
`stellar/anchor-platform:latest` -> `docker pull stellar/anchor-platform:<vesion>` as part of `yarn ap:versions:new` in the future? (It's a nice to have but don't think is mandatory)

Reply:
Good call. I'll rename it manually for now.